lstchain/mc/plot_utils.py

```diff
@@ -331,23 +336,22 @@ def plot_positions_survived_events(df_gammas,
     """
 
     e_reco_g = 10**df_gammas.mc_energy
-    e_reco_p = 10**df_protons.mc_energy
+    e_reco_p = 10 ** df_protons.mc_energy
```

Comment:
Wat?

ereco = 10**mc_energy ?

Reply:
I did not write that code but good catch.

Reply:
This had nothing to do with this PR but I updated it nonetheless to take the reco energy from the dataframe